src/cli.js

```diff
@@ -3,6 +3,64 @@ import argv from 'yargs';
 import { version } from 'json!../package';
 
 
+export function getConfig({useCLI=true, _argv=argv} = {}) {
+  if (useCLI === false) {
```

Comment:
I get how this is only used for testing but I think it could lead to confusion for a caller that is embedding the linter as a library. It would be confusing because their config could be incomplete. It's not a big deal but maybe logging a warning here would help -- something like "a dummy config was generated, you may need to specify a real config." Ideally I'd like to see it throw an error but I guess that won't let you get test coverage. Anyway, just a suggestion.